opencti-platform/opencti-front/src/private/components/events/Incidents.tsx

```diff
@@ -111,6 +113,7 @@ const Incidents: FunctionComponent = () => {
       isSortable: isRuntimeSort,
     },
   };
+  const FABReplaced = isFeatureEnable('FAB_REPLACEMENT');
```

Comment:
```suggestion
  const isFABReplaced = isFeatureEnable('FAB_REPLACEMENT');
```

by convention, simple variables do not start with a capital ; and for clarity I would advise to start boolean vars with "is" for clarity.